homeassistant/components/sensor/__init__.py

```diff
@@ -151,6 +160,25 @@ def capability_attributes(self) -> Mapping[str, Any] | None:
     def state_attributes(self) -> dict[str, Any] | None:
         """Return state attributes."""
         if last_reset := self.last_reset:
+            if (
+                last_reset is not None
+                and self.state_class == STATE_CLASS_MEASUREMENT
+                and not self._last_reset_reported
+            ):
+                self._last_reset_reported = True
+                report_issue = self._suggest_report_issue()
+                _LOGGER.warning(
+                    "Entity %s (%s) with state_class %s has set last_reset. Setting "
+                    "last_reset for entities with state_class other than 'total' or "
+                    "total_increasing is deprecated and will be removed from Home "
```

Comment:
According to the PR description `last_reset` will be ignored when setting state class `"total_increasing"`. What's the truth?